src/run-teamscale-upload.sh

```diff
@@ -22,33 +25,37 @@ fi
 if [ -n "$BRANCH_AND_TIMESTAMP" ]; then
   ARGS+=( "--branch-and-timestamp" "$BRANCH_AND_TIMESTAMP" )
 fi
+if [[ "$MOVE_TO_LAST_COMMIT" == "true" ]]; then
+  ARGS+=( "--move-to-last-commit" )
+fi
 if [ -n "$MESSAGE" ]; then
   ARGS+=( "--message" "${MESSAGE}" )
 fi
 if [ -n "$INPUT" ]; then
   ARGS+=( "--input" "${INPUT}" )
 fi
+if [[ "$INSECURE" == "true" ]]; then
+  ARGS+=( "--insecure" )
+fi
 if [ -n "$TRUSTED_KEYSTORE" ]; then
-  ARGS+=( "--trusted-keystore" "${TRESTED_KEYSTORE}" )
+  ARGS+=( "--trusted-keystore" "${TRUSTED_KEYSTORE}" )
 fi
 
 # Transforming the space separated parameter into multiple single-value parameters. This is necessary, since it is not
 # possible in a Github Action to specify the same parameter multiple times as we do for teamscale-upload.
 for linenumber in $APPEND_TO_MESSAGE; do
```

Comment:
why is this parameter called "linenumber"? IIUC it's one line of the message, not a line number, right?